py3status/modules/getjson.py

```diff
@@ -68,28 +65,21 @@ class Py3status:
     def getjson(self):
         """
         """
-        response = {
-            'cached_until': self.py3.time_in(self.cache_timeout),
-        }
-
         try:
-            resp = self.py3.request(self.url, timeout=self.timeout)
-            status = resp.status_code == 200
-            resp = resp.json()
+            full_text = self.py3.request(self.url, timeout=self.timeout).json()
```

Comment:
The thing is that this isn't the `full_text` it is the json output so calling it `json_data` would be a better name.

It is much better to use appropriate meaningful names that describe what they are.  Reusing a variable for different meanings is a nice way to make code harder to read, prone to bugs etc.